rest/route/middleware_test.go

```diff
@@ -156,48 +158,14 @@ func TestCommitQueueItemOwnerMiddlewarePROwner(t *testing.T) {
 			Enabled: true,
 		},
 	}
-	ctx = gimlet.AttachUser(ctx, &user.DBUser{
```

Comment:
I removed TestCommitQueueItemOwnerMiddlewareProjectAdmin because it was basically the same as this test, but with a different set of permissions. That's more a test of the permissions checking functions than this middleware

Reply:
Should we maybe move it somewhere else instead of deleting it?

Reply:
I don't think so, the logic it's testing is already tested in a different package